src/libgetenv/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 include(CheckSymbolExists)
 check_symbol_exists(__GNU_LIBRARY__ "features.h"    HAVE_GLIBC)
 
-if (BUILD_SHARED AND ENABLE_CXX11 AND HAVE_GLIBC)
+if (BUILD_SHARED AND ENABLE_CXX11 AND HAVE_GLIBC AND ${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
A friend of mine did search for features.h on his OS X (xcode installed) without any results. So is this additional SYSTEM_NAME check really necessary?